build/package/Microsoft.DotNet.Cli.Installer.PKG.targets

```diff
@@ -1,10 +1,132 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    
-    <UsingTask TaskName="PkgTargets" AssemblyFile="$(CLIBuildDll)" />
+    <UsingTask TaskName="ReplaceFileContents" AssemblyFile="$(CLIBuildDll)" />
+  
+    <Target Name="SetupPkgInputsOutputs">
+      <PropertyGroup>
+        <InstallerOutputDirectory>$(PackagesDirectory)</InstallerOutputDirectory>
+        <PkgIntermediateDirectory>$(IntermediateDirectory)/pkgs/$(SdkVersion)</PkgIntermediateDirectory>
 
-    <Target Name="GeneratePkgs" >
-        <PkgTargets />
+        <SdkInstallerFile>$(InstallerOutputDirectory)/$(ArtifactNameWithVersionSdk)$(InstallerExtension)</SdkInstallerFile>
+        <CombinedFrameworkSdkHostInstallerFile>$(InstallerOutputDirectory)/$(ArtifactNameWithVersionCombinedHostHostFxrFrameworkSdk)$(BundleExtension)</CombinedFrameworkSdkHostInstallerFile>
+
+        <!-- Rename is intentional to reduce confusion over generated vs downloaded pkg's -->
+        <DownloadedSharedFrameworkInstallerFile>$(SharedFrameworkInstallerFile)</DownloadedSharedFrameworkInstallerFile> 
+        <DownloadedHostFxrInstallerFile>$(HostFxrInstallerFile)</DownloadedHostFxrInstallerFile>
+        <DownloadedSharedHostInstallerFile>$(SharedHostInstallerFile)</DownloadedSharedHostInstallerFile>
+        
+        <!-- Properties for pkg build -->
+        <SharedHostComponentId>com.microsoft.dotnet.sharedhost.component.osx.x64</SharedHostComponentId>
+        <HostFxrComponentId>com.microsoft.dotnet.hostfxr.component.osx.x64</HostFxrComponentId>
+        <SharedFrameworkComponentId>com.microsoft.dotnet.sharedframework.$(SharedFrameworkNugetName).$(SharedFrameworkVersion).component.osx.x64</SharedFrameworkComponentId>
```

Comment:
I think it has something to do with the SXS, but am honestly not sure. For building these things it should be inconsequential as long as it matches what is in the PKG distribution template. 

Although Now I'm wondering if the Hostfxr pkg has a bug that will break the SxS pkg install. I will investigate this Post this PR. For now this is a 1:1 translation of what is there.